ci/checks/check_integration_test_gets_run.py

```diff
@@ -66,7 +66,7 @@ def check_dependencies(*deps: str) -> None:
 
 def get_test_files(target_dir: pathlib.Path) -> Set[str]:
     try:
-        res_snippets = subprocess.run(
+        res_snippets = subprocess.run(  # noqa: PLW1510
```

Comment:
https://docs.astral.sh/ruff/rules/subprocess-run-without-check/

>By default, subprocess.run does not check the return code of the process it runs. This can lead to silent failures.

>Instead, consider using check=True to raise an exception if the process fails, or set check=False explicitly to mark the behavior as intentional.

I don't want to make runtime changes to our CLI code, since it's mostly(?) deprecated.